AndriyZhornyak/HMW5.py/HMW5.py

```diff
@@ -0,0 +1,33 @@
+# Task1
+list = [221321,534,2324]
+list1 = []
+for item in list:
+    list1.append(float(item))
+print(list1)
+# Task2
+n = 15
+num1 = 0
+num2 = 1
+number3 = num2  
+count = 1
+ 

```

Comment:
good job
look at this implementation
n = int(input('enter some number from fibonacci list= '))

fibonacci_numbers = [0, 1]

for i in range(2,n):
        fibonacci_numbers.append(fibonacci_numbers[i-1]+fibonacci_numbers[i-2])
print(f'there are some numbers from fibanacci till that you entered {n},\n {fibonacci_numbers}')